### DISCUSSION TITLE
What does GitHub student packs time limit means?

### DISCUSSION THREAD

**Discussion by fighteriscool** [community/community#23670] [New to GitHub]
<p>On some of the benefits, there is written something like “Free 12 month subscription of Canva’s Pro tier.” On such benefits,</p>
<p>will the 12 months count start right after I get my GitHub Student Developer Pack. Or when I start to get my that Specific benefits.</p>
<p>Such I got my guthub student developer pack 1st January, 2021 then I want to use canva from 1st march, 2021. Will my benefit on canva end on 1st Jan, 2022 or, 1st march, 2022.</p>
<p>And, Are all the benefits will start right after i get the pack. Or i can start anytime I want?</p>
<p><a href="https://education.github.com/benefits?type=student" rel="noopener nofollow ugc">Get your pack</a></p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>Most of the offers start when activated, not when the Student Pack is approved.</p>
<p>However - the partner offers are entirely controlled by their own company policies. If you have questions about when to activate a specific offer, it would be best to direct it to that company’s support team.</p>

**Comment by ghost** [comment]
i can't activate the canva pro membership
